playwright.config.mjs

```diff
@@ -1,92 +0,0 @@
-// @ts-check
-const { defineConfig, devices } = require('@playwright/test');
-
-/**
```

Comment:
This looked like an auto-generated config to me, with a bunch of defaults. I pared down to whatever defaults are in `defineConfig`. Happy to restore anything